src/monarch/src/bicep.ts

```diff
@@ -20,6 +20,8 @@ const keywords = [
   'in',
   'if',
   'existing',
+  'import',
```

Comment:
>import [](http://example.com/codeflow?start=3&length=6)

Do we need to notify or make updates in some other repos to update grammars? GH Linguist? Others?

(If yes, then we should probably wait until we're done updating syntax, but should probably create a task to track this work.)

Reply:
Good point, I'll create an issue - suggest we probably get around to it once we've finalized the list for the 1.0 release.

Reply:
Created https://github.com/Azure/bicep/issues/4611